packages/blade/src/components/Icons/_Svg/Svg/Svg.web.tsx

```diff
@@ -12,19 +17,17 @@ const Svg = ({
   ...styledProps
 }: SvgProps): ReactElement => {
   return (
-    // line-height adds some extra spacing in div around svg so removing it
-    <BaseBox display="inline-block" lineHeight="spacing.0" {...getStyledProps(styledProps)}>
-      <svg
-        {...makeAccessible({ hidden: true })}
-        {...metaAttribute(MetaConstants.Component, MetaConstants.Icon)}
-        height={height}
-        viewBox={viewBox}
-        width={width}
-        fill={fill}
-      >
-        {children}
-      </svg>
-    </BaseBox>
+    <StyledSvg
+      {...styledProps}
+      {...makeAccessible({ hidden: true })}
```

Comment:
Why did we make all our icons hidden by default? @anuraghazra saw your commit for this. Can't recall why we did this

Reply:
Icon's are "Presentational" elements they do not need to be announced to the screen readers, if the icons are interactive or clickable then that's a different matter, but for that we have IconButton.